### DISCUSSION TITLE
⛔ Warning (org-element): ‘org-element-at-point’ cannot be used in non-Org buffer #<buffer  *temp*> (fundamental-mode)

### DISCUSSION THREAD

**Discussion by danielpopescu** [minad/marginalia#181] [Q&A]
Using latest emacs 30.1, macos X, native compilation, with latest packages (org-mode 9.7.9) and latest Marginalia.

When in org-mode, "M-x" produces the warning above.

The only way to get rid of it, it is to comment out my marginalia usage (the code below):

 ; Enable rich annotations using the Marginalia package
(use-package marginalia
  ;; Bind `marginalia-cycle' locally in the minibuffer.  To make the binding
  ;; available in the *Completions* buffer, add it to the
  ;; `completion-list-mode-map'.
  :bind (:map minibuffer-local-map
         ("M-A" . marginalia-cycle))
  ;; The :init section is always executed.
  :init
  ;; Marginalia must be actived in the :init section of use-package such that
  ;; the mode gets enabled right away. Note that this forces loading the
  ;; package.
  (marginalia-mode))

Thank you

**Comment by minad** [comment]
This issue has absolutely nothing to do with Marginalia. Marginalia doesn't interact or interfere with Org in any way. I also use the newest Emacs 30 and the newest Org without issues. It could be that you have some other mode enabled in your configuration which leads to an interaction. Maybe one of your other packages is outdated, e.g., Embark? Before reporting an issue, please try to reproduce the problem with emacs -Q and try to narrow it down.

**Comment by danielpopescu** [comment]
I have all latest everything (including built in packages: (setq package-install-upgrade-built-in 't)).

I commented out (bisect methods) diff parts of my config (including embark) but the only package that makes that go away it is marginalia...

Note, same config it seems to wok fine with latest emacs 29.4 on linux.

**Comment by danielpopescu** [comment]
removed org & re-installed /force recompilation and the problem went away....
